packages/nextjs/components/batch/BuildersGrid.tsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useState } from "react";
+import Link from "next/link";
+import { Address } from "~~/components/scaffold-eth";
+
+interface Builder {
+  id: number;
+  address: string;
+}
+
+// This needs to be in a server component or API route
+// async function getBuilderAddresses() {
+//   const buildersPath = path.join(process.cwd(), 'app/builders');
+//   try {
+//     const directories = await readdir(buildersPath, { withFileTypes: true });
+//     return directories
+//       .filter(dirent => dirent.isDirectory())
+//       .map(dirent => dirent.name)
+//       .filter(name => name.startsWith('0x'));
+//   } catch (error) {
+//     console.error('Error reading builders directory:', error);
+//     return [];
+//   }
+// }
```

Comment:
You chose to create an api route, so this could be removed, right?

Reply:
Yeah, thanks, I will do just that, I forgot that it was still there, I used it to test it before moving it to api route